src/Pages/Profile/MemberView/InfoCard.js

```diff
@@ -9,118 +9,63 @@ const pic = require('./getPicBySeason');
 const bcrypt = require('bcryptjs');
 
 export default function ProfileCard(props) {
-  const [password, setPassword] = useState('New Password');
-  const [confirmPass, setConfirmPass] = useState('Confirming New Password');
-  const [user, setUser] = useState('');
-  const discordLogo =
-    'https://www.freeiconspng.com/uploads/discord-black-icon-1.png';
+	const [password, setPassword] = useState('New Password');
+	const [confirmPass, setConfirmPass] = useState('Confirming New Password');
+	const [user, setUser] = useState('');
+	const [toggle, setToggle] = useState(false);
 
-  async function changePassword() {
-    // hash pass
-    const salt = bcrypt.genSaltSync(10);
-    const hashedPassword =
-        password.trim() === '' ?
-          user.password :
-          bcrypt.hashSync(password, salt);
+	function itemClicked() {
+		// } else if (id === 7) {
+		// 	props.handleDiscordAuth();
+		// } else {
+		// 	setState({
+		// 		...state,
+		// 		[id]: {
+		// 			clicked: state[id] ? !state[id].clicked : true
+		// 		}
+		// 	});
+		// }
```

Comment:
Delete commented lines. Also, the spacing on this file is really weird again. Reformat, save and push again